owners/OWNERS.example

```diff
@@ -0,0 +1,124 @@
+
+        // Adding `notify: true` to an owner definition will ensure that the
+        // user is notified of any PR touching those files, even if they are not
+        // added as a reviewer. The bot will add a comment with an @mention to
+        // these PRs to notify the owners.
+        { name: "rcebulko", notify: true },
+
+        // Note that these options may not be combined; both will be ignored for
+        // the owner below.
+        { name: "rando", notify: true, requestReviews: false },
+      ]
+    },
+
+    {
+      // Each rule may optionally contain a `pattern` key. If present, the rule
+      // will apply only to files matching the pattern. By default, this pattern
+      // applies only to files within the directory (not subdirectories). This
+      // may be used to list filenames explicitly.
+      pattern: "package.json",
+      owners: [{ name: "packager" }],
+    },
+
+    {
+      // Glob-like patterns may be used in place of literal filenames. In these
+      // patterns, a star `*` can represent any sequence of characters. These
+      // patterns follow the same rules as above, so `*.js` in this directory
+      // applies to `./main.js` but not `./foo/bar.js`.
+      pattern: "*.js",
+      owners: [{ name: "scripter" }],
+    },
+
+    {
+      // To apply a filename or pattern to all subdirectories as well, prefix the
+      // pattern with `**/`. The pattern `**/.eslintrc` applies to `./.eslintrc`
+      // as well as `./foo/.eslintrc`. The pattern `**/*.protoascii` applies to
+      // `./test.protoascii` as well as `./foo/example.protoascii`.
+      pattern: "**/*.protoascii",
+      owners: [{ name: "ampproject/wg-caching" }],
+    },
+
+    {
+      // Any other use of `/` in a pattern is forbidden (ie. no directory
+      // traversal). Patterns using `/` will be ignored. If rules are needed in a
+      // subdirectory, the appropriate OWNERS file should be created/updated in
+      // that subdirectory. This rule will be ignored since the pattern is
+      // illegal.
+      pattern: "foo*/*.js",
+      owners: [{ name: "doesntmatter" }],
+    },
+
+    {
+      // Filename and pattern rules may be combined with brace expansion. The 
+      // pattern parser uses
+      // `[minimatch](https://www.npmjs.com/package/minimatch)`, the matching
+      // library used internally by npm. The pattern `*.{js,css}` applies to 
+      // `./script.js` and `./style.css`.
+      pattern: "*.{js,css}",
+      owners: [{ name: "frontend" }],
+    },
+  ]
+}
+
+// The result of parsing this file would be a tree with these rules:
+//
+// **/*: someuser, dontdothis, ampproject/wg-cool-team [members...],
+//       dvoytenko (never notify), rcebulko (always notify), rando
+// ./{package.json}: packager
+// ./{*.js}: scripter
+// **/{*.protoascii}: ampproject/wg-caching [members...]
+// ./{*.{js,css}}: frontend
+// ./{package.json,yarn.lock}: ampproject/wg-infra [members...]
```

Comment:
In addition to this and the comprehensive list of examples above it, I think it would be useful to include an example of a canonical "hello world" owners file with just one or two entries at the top of the file. WDYT?

Reply:
Added a simple example in the fileoverview